lib/components/Pagination/Pagination.docs.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { ComponentDocs } from '../../../site/src/types';
+import { Pagination, Card } from '../';
+import source from '../../utils/source.macro';
+import { Strong } from '../Strong/Strong';
+import { Text } from '../Text/Text';
+import { TextLink } from '../TextLink/TextLink';
+import { maxPages } from './paginate';
+
+const docs: ComponentDocs = {
+  category: 'Content',
+  Example: ({ setDefaultState, getState, setState }) =>
+    source(
+      <>
+        {setDefaultState('page', 5)}
+
+        <Card>
+          <Pagination
```

Comment:
Should we show a stack of examples here in various states? I feel like it would provide a quicker overview of the component from a design standpoint.